config/query-api.yml

```diff
@@ -108,3 +109,24 @@ definitions:
       - title
 
 
+  SubscriptionData:
+    type: object
+    properties:
+      transaction_id:
+        type: string
+      subscription_id:
+        type: string
+        description: ID of DSS subscription
+      es_query:
```

Comment:
It doesn't seem that you use `transaction_id`, `subscription_id`, and the `es_query` anywhere. You can probably remove these validation fields, then.

To my understanding, `match` is the only thing that is really important to validate here.

Reply:
👍